packages/babel-preset-app/src/index.js

```diff
@@ -55,6 +55,7 @@ module.exports = (api, options = {}) => {
   const envName = api.env()
 
   const {
+    bugfixes,
```

Comment:
What do you think of setting explicit default value here?

Reply:
I think keeping same default value as babel make it default true until babel 8 is proper.

Reply:
I can add false to it. And change to true when upgrade to babel 8

Reply:
Both LGTM. I let you decide state to merge :)